### ISSUE TITLE
Training epoch

### ISSUE THREAD

**Issue by Xyhi** [usail-hkust/t-PatchGNN#4]
Dear authors, I really appreciate your amazing work. May I ask if the training epochs are all set as 1000 in the experiments?

**Comment by willzhang3** [comment]
All models apply the same early stopping strategy when the validation loss doesn’t decrease over 10 epochs. Therefore, the total training epochs may vary for different models and usually less than 1000.
